assets/src/scripts/d3-rendering/tooltips.spec.js

```diff
@@ -1,8 +1,9 @@
 import {select} from 'd3-selection';
-import {appendTooltip} from 'ui/tooltips';
 
+import {appendTooltip} from './tooltips';
 
-describe('tooltips', () => {
+
+describe('d3-rendering/tooltips', () => {
```

Comment:
Is just to tell us which tooltips are being tested, or is there greater significance?

Reply:
Yeah. I have been trying to change these so it's easier in the test output to figure out where the error occurred. In this particular case we have tooltips in d3-rendering and the then tooltip in hydrograph and dv-hydrograph.